rpcs3/Emu/RSX/GL/GLGSRender.cpp

```diff
@@ -1565,21 +1565,16 @@ void GLGSRender::flip(int buffer, bool emu_flip)
 	u32 buffer_pitch = display_buffers[buffer].pitch;
 
 	u32 av_format;
-	const auto avconfig = fxm::get<rsx::avconf>();
+	const auto avconfig = g_fxo->get<rsx::avconf>();
 
-	if (avconfig)
+	if (true)
```

Comment:
Its possible to reach here before the game has initialized at which point we want to not use the default values in avconf at all. Maybe we can have an extra argument like initialized or something to skip this part if cellVideoOutConfigure is yet to be called.